common/app/views/support/MinuteCleaner.scala

```diff
@@ -101,5 +101,7 @@ case class MinuteCleaner(article: model.Article) extends HtmlCleaner {
 }
 
 object regexCleaner {
-  def apply(heading: Element, regEx: String, htmlToReplace: String): String = heading.html().replaceFirst(regEx, htmlToReplace)
+  def apply(heading: Option[Element], regEx: String, htmlToReplace: String): String = heading.map { heading =>
+    heading.html().replaceFirst(regEx, htmlToReplace)
```

Comment:
Is this clearer?
    
    heading
      .map(_.html)
      .getOrElse("")
      .replaceFirst(regEx, htmlToReplace)